package.json

```diff
@@ -34,8 +42,10 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-react-app": "^3.1.0",
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.24.1",
+    "babel-runtime": "6.26.0",
```

Comment:
Do we need to change the semver to allow for picking up new versions?  `babel-runtime` is currently fixed to version `6.26.0`